app/api/V2/models/parcelmodels.py

```diff
@@ -30,15 +30,17 @@ def serialize(self):
             curr_location=self.curr_location
         )
 
-    def get_by_id(self, order_id):
-        """Get a single item by unique id."""
-        for order in orders:
-            if order.id == order_id:
-                return order
+    @classmethod
+    def get_order_by_orderid(cls, order_id):
```

Comment:
This method name can be improved to `get_order_by_order_id`. This will make it to be more readable.